proto/v1alpha1/rode.proto

```diff
@@ -72,6 +72,12 @@ service Rode {
     };
     option (google.api.method_signature) = "filter";
   }
+  rpc ValidatePolicy(ValidatePolicyRequest) returns (ValidatePolicyResponse) {
+    option (google.api.http) = {
+      post: "/v1alpha1/policies/validate"
```

Comment:
I don't feel strongly about it, but for the RPC calls that don't map to restful endpoints (like attest policy or batch create) we've been using a colon to append the action, like `/v1alpha1/policies:validate"`. That seems to be the convention that Grafeas follows at least.

Reply:
@pickjasmine This change is gonna affect your PR i assume. just a heads up